float64[] angles
float64[] gpos
float64[] vel
float64[] gvel
float64[] torque
float64[] gforce
